.github/CODEOWNERS

```diff
@@ -1,15 +1,16 @@
-/docs/       @Jigsaw-Code/outline-strings-owners
-/resources/  @Jigsaw-Code/outline-strings-owners
-*.json       @Jigsaw-Code/outline-strings-owners
-*.md         @Jigsaw-Code/outline-strings-owners
-*.png        @Jigsaw-Code/outline-strings-owners
-*.svg        @Jigsaw-Code/outline-strings-owners
+/docs/       @Jigsaw-Code/outline-releasers
+/resources/  @Jigsaw-Code/outline-releasers
+*.json       @Jigsaw-Code/outline-releasers
+*.md         @Jigsaw-Code/outline-releasers
+*.png        @Jigsaw-Code/outline-releasers
+*.svg        @Jigsaw-Code/outline-releasers
+
+/src/cordova/             @Jigsaw-Code/outline-releasers
+/src/electron/            @Jigsaw-Code/outline-releasers
+/cordova-plugin-outline/  @Jigsaw-Code/outline-releasers
```

Comment:
Does this work?
```suggestion
*       @Jigsaw-Code/outline-dev
```

Reply:
Not sure - let's see

Reply:
Seems like it. I thought maybe by https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners you were supposed to use `/` instead of `*` but github said it wasn't a valid CODEOWNERS file